aif360/algorithms/inprocessing/gerryfair/reg_oracle_class.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+
+class RegOracle:
+    """Class RegOracle, linear threshold classifier."""
+    def __init__(self, b0, b1):
+        self.b0 = b0
+        self.b1 = b1
+
+    def predict(self, X):
+        """Predict labels on data set X."""
+        reg0 = self.b0
+        reg1 = self.b1
+        n = X.shape[0]
+        y = []
+        for i in range(n):
+            x_i = X.iloc[i, :]
+            x_i = x_i.values.reshape(1, -1)
+            c_0 = reg0.predict(x_i)
```

Comment:
Is there a reason why `predict` can't work on all the samples at once, i.e. `reg0.predict(X)`?